### ISSUE TITLE
Add more Items

### ISSUE THREAD

**Issue by DoganM95** [openhab/openhab-google-assistant#163]
The list of possible items so far is not bad, but google has added lots of possible devices of which only a fraction is implemented in openhab, listed here:
[https://developers.google.com/assistant/smarthome/guides](https://developers.google.com/assistant/smarthome/guides) 

Each item also supports multiple actions like an oven with the following actions:
Cook, Modes, OnOff, StartStop, TemperatureControl, Toggles.

Could you please update the binding to include the remaining device types and their actions?

**Comment by michikrug** [comment]
Hi. It is not as easy as just "adding" them. We will try the best to add more and more device types over time. But more different interactions also mean more implementation effort.

**Comment by eric1905** [comment]
I would be very interested in https://developers.google.com/assistant/smarthome/guides/audiovideoreceiver
AppSelector and InputSelector looks like a good opportunity to control my enigma2 receiver or my tv by voice.

**Comment by t3h-n00b** [comment]
> I would be very interested in https://developers.google.com/assistant/smarthome/guides/audiovideoreceiver
> AppSelector and InputSelector looks like a good opportunity to control my enigma2 receiver or my tv by voice.

Yes, would be really good to get a TV-like item in there - It is easy enough to turn on/off the system with a switch in Google Assistant; but volume+/-, input/channel change etc. via Google would be great as well

**Comment by michikrug** [comment]
I have started the implementation of TV - currently Volume, Channel, Input and TransportControl are working

**Comment by goofy1988** [comment]
Wow, that is great. This feature will raise this binding to a whole new/next level for me. Thanks alot for your effort.

**Comment by SEQUOiA87** [comment]
> I have started the implementation of TV - currently Volume, Channel, Input and TransportControl are working

@michikrug  Any news on the TV implementation? Did you stop working on it? Is it much effort or are there any technical showstopper? Would love to see my TV controls represented as one device within the Google Home app :)

**Comment by michikrug** [comment]
@SEQUOiA87 I paused further testing in favor of the special color light and temperature sensor implementation

But there are no issues to name so far

**Comment by SeveranExp** [comment]
Why not make a monthly poll for openhab users to vote on what items they would like you guys to push forward on?
That would be interesting, get more engagement from the community will also spark more enthusiasm.

**Comment by michikrug** [comment]
btw @SEQUOiA87
I added a bit of documentation on the TV implementation here: https://github.com/openhab/openhab-google-assistant/pull/175/files#diff-52296615debf3994195e43d91c94e02bfdc615a27620e0dbc72140b3de98dde1R58
